src/thor/dijkstras.cc

```diff
@@ -235,10 +235,10 @@ void Dijkstras::Expand(ExpansionType expansion_type,
       Compute(*api.mutable_options()->mutable_locations(), reader, costings, mode);
       break;
     case ExpansionType::reverse:
-      Compute(*api.mutable_options()->mutable_locations(), reader, costings, mode);
+      ComputeReverse(*api.mutable_options()->mutable_locations(), reader, costings, mode);
```

Comment:
Is it possible to add tests (new or update existing) to catch such issues? (haven't looked at current isochrone tests)

Reply:
its hard to do the multimodal one unless we just test that it works for a short pedestrian only route. one day we need to figure out how to generate transit pbfs from gurka that would be awesome.

as for reverse we can totally test it but it will be just as unscientific of a test as the existing isochrone test in that we'll just check that a polygon comes out. its very hard to really accurately test this type of thing where the result is a geometric shape.